app/assets/stylesheets/application/footer.scss

```diff
@@ -5,8 +5,14 @@
 /* Footer */
 
 #footer {
-	background-color: $secondary-background-color;
+	// background: $secondary-background-color;
 	padding: 40px 0px;
+	/* Gradient */
+	left: 0px;
+	top: 0px;
+
+	background: linear-gradient(142.51deg, #700F3C 62.56%, #D0E799 204.67%);
```

Comment:
I see that you re-defined `$secondary-background-color` as this. Why not use that?